src/app/pages/tabs-subpage/tabs-subpage.component.ts

```diff
@@ -0,0 +1,19 @@
+import { CUSTOM_ELEMENTS_SCHEMA, Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-tabs-subpage',
+  standalone: true,
+  templateUrl: './tabs-subpage.component.html',
+  styleUrls: ['./tabs-subpage.component.css'],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export class TabsSubpageComponent {
+  currentRoute: string;
+
+  constructor(private route: ActivatedRoute) {
+    this.route.url.subscribe((segments) => {
```

Comment:
above comment applies :) 

you can mark it as resolved when you read here :)  it is not a blocker.